examples/how-to-use-webrtc-star/README.md

```diff
@@ -0,0 +1,14 @@
+# How to use WebRTC star
+
+There's still a bit of work required to start up an in-browser node in a robust way, so that it will work whether or not there is an existing initialized IPFS repo in the user's browser. If there isn't one, you need to call `init` as above, but if there is one, calling `init` will fail. Moreover, there's currently no good way to check if you need to call `init` or not.
+
+Also, an in-browser node isn't able to call up normal IPFS nodes over raw TCP; it can only communicate over Websockets and WebRTC. Currently, there are no Websockets or WebRTC bootstrap nodes run by the IPFS maintainers. You will probably want to set up a [libp2p-webrtc-star signaling server](https://github.com/libp2p/js-libp2p-webrtc-star) so nodes used in your application can find each other:
```

Comment:
We do run a publicly available signaling server which would better to point users to rather than everyone setting up their own. We should start with an explanation for connecting to the existing one and later also explain how to run their own if necessary.